src/app/utils/app-utils.ts

```diff
@@ -94,31 +95,4 @@ export class AppUtils {
 
     return organisations;
   }
-
-  public static capitalizeString(stringToCapitalize: string) {
```

Comment:
not sure if/when this block of code went in, but ensure it isn't needed

Reply:
@paddybasi  just verified both functions are not being used anywhere. Thanks.